docs/general/builtins/scalar-functions.rst

```diff
@@ -52,6 +52,12 @@ You can also use the ``||`` :ref:`operator <gloss-operator>`::
     +--------+
     SELECT 1 row in set (... sec)
 
+.. NOTE::
+
+    The ``||`` operator differs from the ``concat`` function in that it
```

Comment:
`in that it...` reads a bit weird to me. Although it is much less compact, hoow about:
```
The ``||`` operator differs from the ``concat`` function regarding handling of ``NULL`` arguments. the ``||`` operator will ...
```